boards/arm/frdm_k64f/dts.fixup

```diff
@@ -29,3 +29,12 @@
 #define CONFIG_WNCM14A2A_GPIO_MDM_SEND_OK_NAME		NXP_KINETIS_UART_4006C000_WNCM14A2A_MDM_SEND_OK_GPIOS_CONTROLLER
 #define CONFIG_WNCM14A2A_GPIO_MDM_SEND_OK_PIN		NXP_KINETIS_UART_4006C000_WNCM14A2A_MDM_SEND_OK_GPIOS_PIN
 #endif
+
+#if ETH_PORT == 0
```

Comment:
Remove the #if ETH_PORT check, we tend to have the fixup always, if there isn't any reason not to.